login.html

```diff
@@ -0,0 +1,147 @@
+                        preserveAspectRatio="xMidYMid meet" viewBox="0 0 24 24">
+                        <g fill="none">
+                            <path
+                                d="M3 3h2l.4 2M7 13h10l4-8H5.4M7 13L5.4 5M7 13l-2.293 2.293c-.63.63-.184 1.707.707 1.707H17m0 0a2 2 0 1 0 0 4a2 2 0 0 0 0-4zm-8 2a2 2 0 1 1-4 0a2 2 0 0 1 4 0z"
+                                stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round">
+                            </path>
+                        </g>
+                    </svg></a>
+
+                <a class="homepage-nav-links" href="./wishlist.html" target="_blank"><svg xmlns="http://www.w3.org/2000/svg"
+                        xmlns:xlink="http://www.w3.org/1999/xlink" aria-hidden="true" role="img"
+                        class="iconify iconify--mdi" width="32" height="32" preserveAspectRatio="xMidYMid meet"
+                        viewBox="0 0 24 24">
+                        <path
+                            d="M12.1 18.55l-.1.1l-.11-.1C7.14 14.24 4 11.39 4 8.5C4 6.5 5.5 5 7.5 5c1.54 0 3.04 1 3.57 2.36h1.86C13.46 6 14.96 5 16.5 5c2 0 3.5 1.5 3.5 3.5c0 2.89-3.14 5.74-7.9 10.05M16.5 3c-1.74 0-3.41.81-4.5 2.08C10.91 3.81 9.24 3 7.5 3C4.42 3 2 5.41 2 8.5c0 3.77 3.4 6.86 8.55 11.53L12 21.35l1.45-1.32C18.6 15.36 22 12.27 22 8.5C22 5.41 19.58 3 16.5 3z"
+                            fill="currentColor"></path>
+                    </svg></a>
+
+                    <a class="homepage-nav-links" href="./login.html" target="_blank">
+                        <div>LOGIN</div>
+                    </a>
+
+                <!-- Avatar -->
+                <a href="#">
+                    <div class="avatar avatar-sm">
+                        <img src="../Images/pingu.jpg" alt="penguin-w-headphones">
+                    </div>
+                </a>
+            </div>
+        </div>
+        <!-- navbar ends here -->
+        
+        <!-- modal starts here -->
+        <div class="modal-container">
+            <div class="modal-card">
+                <div class="inner-box" id="modal-card">
+                    <div class="card-front">
+                        <h2>LOGIN</h2>
+                        <form>
+                            <input type="email" class="form-input-box" placeholder="Enter your Email id" required>
+                            <input type="password" class="form-input-box" placeholder="Password">
+                            <button type="submit" class="submit-btn">
+                                Submit
+                            </button>
+                            <div class="checkbox-remember-me">
+                                <input type="checkbox"><span>Remember me</span>
+                            </div>
+                        </form>
+                        <button type="button" class="btn-new-here" onclick="openRegister()">I'm New Here</button>
```

Comment:
The login/signup page is looking superb. Animation is very subtle and amazing looking. Just ignore to put javascript code in the HTML file; place it inside a separate javascript file.  There is no hard rule but we should separate logic and skeleton codes in a separate file. Other than that everything is great. Awesome work.